Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -82,9 +82,17 @@ class SyncNowSetting: WithAccountSetting {
         return formatter
     }()
 
-    private let syncNowTitle = NSAttributedString(string: NSLocalizedString("Sync Now", comment: "Sync Firefox Account"), attributes: [NSForegroundColorAttributeName: UIColor.blackColor(), NSFontAttributeName: DynamicFontHelper.defaultHelper.DefaultStandardFont])
+    private var syncNowTitle: NSAttributedString {
+        return NSMutableAttributedString(
```

Comment:
if we never mutate this string it doesn't need to be a `NSMutableAttributedString`